### DISCUSSION TITLE
Difficulty level of driving

### DISCUSSION THREAD

**Discussion by mr-ukas** [StuckAtPrototype/Racer#27] [General]
Hi, I love your project idea. I am curious about the overall user experience of driving? 

Without a mounted camera module and VR-glasses, I guess steering the racer is pretty hard, especially if your POV is from the otherside of the driving direction. 

How old are your kids? Do they manage to drive the cars along a little parcour/track?

**Comment by StuckAtPrototype** [comment]
Hi! My 4 year old is able to drive them around with fair precision, though id recommend 4 as the absolute minimum age. I am able to drive them around on a desk, etc. The driving mechanism could definitely be greatly improved. One way would be to use the analog joystick for both maneuvering and speed control.

**Reply by DapperDan81** [reply]
Hi,

did you notice any differences between the four motors? Is the final car driving in a straight line or does it tend to go to the left ore right slightly?
I have seen a similar project on youtube also with four motors and that one was not able to go in a straight line.
https://youtu.be/gmqgAcnKeFU?feature=shared

**Comment by mr-ukas** [comment]
Thanks for the honest feedback. 
Does the design allow mounting a camera module? But that's probably a topic for itself..
